GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -246,27 +260,144 @@ void beforeDrawEyes() {
         mFrameHandler.beforeDrawEyes();
     }
 
+    @Override
+    public void captureScreenCenter(GVRScreenshotCallback callback) {
+        mScreenshotCenterCallback = callback;
```

Comment:
I guess it's 'harmless' to not check for `null` since you are using the callback as a flag, but I think you really should throw an `IllegalArgumentException` when `callback == null`